public/manifest-assets/html_code.html

```diff
@@ -0,0 +1,9 @@
+<link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
```

Comment:
What is this? looks like copypasta

Reply:
https://developer.apple.com/library/archive/documentation/AppleApplications/Reference/SafariWebContent/ConfiguringWebApplications/ConfiguringWebApplications.html